packages/modular-scripts/src/__tests__/utils/config.test.ts

```diff
@@ -13,28 +13,45 @@ const configFixtures = path.join(modularRoot, '__fixtures__', 'test-config');
 // Temporary test context paths set by createTempModularRepoWithTemplate()
 let tempModularRepo: string;
 
-describe('A simple modular repo with a .modular.js config file', () => {
+const app = 'test-app';
+const esbuildConfigFile = 'esbuild-config.js';
+const webpackConfigFile = 'webpack-config.js';
+
+describe('A modular repo with a root .modular.js config file', () => {
   beforeEach(() => {
     tempModularRepo = createModularTestContext();
-    runModularForTests(tempModularRepo, 'add test-app --unstable-type app');
+    runModularForTests(tempModularRepo, `add ${app} --unstable-type app`);
     copyFileSync(
-      path.join(configFixtures, '.modular.js'),
-      path.join(tempModularRepo, 'packages', 'test-app', '.modular.js'),
+      path.join(configFixtures, esbuildConfigFile),
+      path.join(tempModularRepo, '.modular.js'),
     );
   });
-  it('builds using esbuild as specified in config file', () => {
+  it('builds using esbuild as specified in the root config file', () => {
     const result = runModularPipeLogs(
       tempModularRepo,
-      `build test-app --verbose`,
+      `build ${app} --verbose`,
       'true',
     );
     expect(result.stdout).toContain('Building with esbuild');
     expect(result.exitCode).toBe(0);
   });
+  it('builds using webpack if a package specific config gile overrides the root config', () => {
```

Comment:
I'm not sure if this is really testing the case where a package is overwriting the global config, since you're only importing the webpack config. Maybe it would be better to rename the config files to "root.config.js" and "package.config.js", import both and check that the package one is being picked up only?

Reply:
So in this specific test you have:
-  .modular.js in the root specifying to use esbuild (added in the beforeEach code)
- .modular.js in the package specifying use webpack 

The test is, if it builds with webpack, the package config has overridden the root config (in the previous test case, there is only a root config, and we check that it builds with esbuild)

I feel like this covers it? unless I'm misunderstanding what you're saying

Reply:
oh I think you're right! The base config still exists from the previous test and this is overwriting it

Reply:
Yep the root config is added in all 3 tests thought the beforeEach